docs/contributor/05-20-e2e_tests.md

```diff
@@ -57,3 +57,23 @@ The GitHub Actions workflows execute the two tests:
 The Kubernetes cluster is created, and the sources are checked out.
 The workflows wait till the binary image is available for fetching.
 The scripts create the required prerequisites, get the BTP Manager and BtpOperator installed or upgraded, validate expected statuses, and get BtpOperator and BTP Manager uninstalled.
+
+### Real credentials rotation
+
+Real credentials used in the test are configured as repository secrets.
+Following secrets are used and substituted in the `sap-btp-manager` secret resource:
+- SM_CLIENT_ID - Service Manager client ID, `data.clientid`
+- SM_CLIENT_SECRET - Service Manager client secret, `data.clientsecret`
+- SM_URL - Service Manager URL, `data.sm_url`
+- SM_TOKEN_URL - Service Manager token URL, `data.tokenurl`
+All secrets should be base64 encoded. Caveat of the new line character at the end of the secret value.
+
+Bash command
+
+```echo -n "secret" | base64``` 
+
+could be used to encode the secret.
+
+Currently used values are taken from Service Binding `e2e-test-sm` created for `e2e-test-sm` Service Manager instance in `e2e-test-btp-manager` subaccount of `kyma-gopher` global account on Canary environment.
+In case of credentials rotation, the secrets should be updated in the repository secrets regardless of location and naming of Service Manager instance and Secret Binding used.
```

Comment:
```suggestion
In case of credentials rotation, the secrets should be updated in the repository secrets, regardless of the location and naming of the Service Manager instance and Secret Binding used.
```